docs/subquery_network/node_operators/maximise-apy.md

```diff
@@ -0,0 +1,171 @@
+  "indexer": "",
+  "os": "Debian GNU/Linux 10 4-CPU",
+  "proxyVersion": "v2.4.2",
+  "uptime": 8066
+}
+```
+- Ensure you have at least one flexplan enabled for each project.
+- Ensure projects are able to fetch metadata from deployment’s Query Service. An error will appear in the status column if there is an issue.
+
+## Selecting what projects to run.
+
+As a node operator, three types of projects can be run: SubQuery data indexers, Subgraphs, or RPC endpoints. See https://app.subquery.network/explorer/home for a complete list. 
+
+Rewards are generated by each project based on the amount of boost allocated to a project and then shared proportionally between the number of operators (for that project).
+
+What to look out for? Projects with:
+- high boost and therefore high stake rewards
+- high query numbers
+- low competition
+- low project complexity
+
+Project database size is a good indicator of cost
+
+## Node Operator Rewards FAQs
+
+### Which projects should I run to generate the highest rewards?
+
+Project rewards come from a combination of its Stake Rewards and Query Rewards. 
+
+Stake rewards are measured in stake APY, the Annual Percentage Yield you can expect from allocating SQT to a project. It is a function of the boost applied to a project, which sets the size of its total stake reward pool, divided by the stake allocated to the project. 
+
+To maximise stake APY, look for projects which have high boost, but low allocated stake.
+
+Query Rewards are generated when people pay for queries against a project. 
+
+To maximise query rewards look for projects that have high average query rewards per Node Operator.
+
+### Should I run Index projects or RPC projects?
+
+Index projects typically receive a higher portion of their rewards via stake rewards which is relatively stable era-to-era. 
+
+RPC projects typically receive more query rewards which can produce a higher overall APY but with more volatility as query demand changes.
+
+Running RPCs is very easy if you are already familiar with them, and it’s suggested to connect your existing RPCs to the SubQuery Network if you are already running an RPC for another usecase (the RPC service does not need to be dedicated to the SubQuery Network).
+
+### How can I attract more RPC requests?
+
+The public RPC gateway will allocate RPC requests based on your:
+- Reliability: Improve your uptime to increase queries.
+- Block height delay: Keep your nodes fully synced to receive queries.
+- Latency: Reduce latency by keeping your Node CPU usage healthy with good network connection.
+- Price: Lower pricing can receive more RPC requests.
+
+### Should I run all RPC projects?
+
+If you are already running RPC nodes which have capacity to serve more requests, it may be beneficial to add SubQuery as an extra source of requests to maximise your return from the node with very little added cost.
+
+### Do I need to allocate stake to my RPC projects to earn query rewards?
```

Comment:
Should we explain more about the why rather than using the indexer/RPC differentiator. Root cause etc?

For example some indexer projects will rely much more in stake rewards than query rewards? How do you know when and what should you do in either cases